metron-stellar/stellar-common/src/main/antlr4/org/apache/metron/stellar/common/generated/Stellar.g4

```diff
@@ -272,4 +278,30 @@ lambda_variable:
   IDENTIFIER
   ;
 
+match_expr :
+  MATCH LBRACE match_clauses RBRACE #MatchClauses
+  ;
+
+match_clauses :
+  match_clause (COMMA match_clause)*
+  ;
+
+match_clause :
+  match_clause_check COLON  match_clause_action
+  ;
+
+match_clause_action :
+  match_clause_action_expr #MatchClauseAction
+  ;
 
+match_clause_action_expr :
+  transformation_expr
+  ;
+
+match_clause_check :
```

Comment:
Is this extra expression needed? Could we get rid of it? Instead of this just have what's below?

```
match_clause_action :
  transformation_expr #MatchClauseAction
  ;

match_clause_check :
  logical_expr #MatchClauseCheckExpr
  ;
```

Reply:
I'll try that again.  I don't remember the specific reason why I ended up with it factored like this.  But this is what ended up working ;)

Reply:
Done